app/split_integer.py

```diff
@@ -1,7 +1,5 @@
 def split_integer(value: int, number_of_parts: int) -> list:
-    parts = []
-    for parts_left in range(number_of_parts, 0, -1):
-        next_number = value // parts_left
-        parts.append(value // parts_left)
-        value -= next_number
-    return parts
+    result_list = [value // number_of_parts] * number_of_parts
```

Comment:
as far as I get it, we`re not supposed to change split_integer.py
please, ask a mentor about it